socorro/scripts/fetch_crash_data.py

```diff
@@ -69,55 +73,53 @@ def fetch_crash(outputdir, api_token, crash_id):
 
     raw_crash = resp.json()
 
-    # Fetch dumps
-    dumps = {}
-    dump_names = raw_crash.get('dump_checksums', {}).keys()
-    for dump_name in dump_names:
-        print('Fetching %s -> %s' % (crash_id, dump_name))
-        if dump_name == 'upload_file_minidump':
-            dump_name = 'dump'
-
-        resp = requests.get(
-            HOST + '/api/RawCrash/',
-            params={
-                'crash_id': crash_id,
-                'format': 'raw',
-                'name': dump_name
-            },
-            headers=headers,
-        )
-
-        if resp.status_code != 200:
-            raise Exception('Something unexpected happened. status_code %s, content %s' % (
-                resp.status_code, resp.content)
-            )
-
-        dumps[dump_name] = resp.content
-
-    # Save everything to file system in the right place
-    # http://antenna.readthedocs.io/en/latest/architecture.html#aws-s3-file-hierarchy
-
     # Save raw crash to file system
     fn = os.path.join(outputdir, 'v2', 'raw_crash', crash_id[0:3], '20' + crash_id[-6:], crash_id)
     create_dir_if_needed(os.path.dirname(fn))
     with open(fn, 'w') as fp:
         json.dump(raw_crash, fp, cls=JsonDTEncoder, indent=2, sort_keys=True)
 
-    # Save dump_names to file system
-    fn = os.path.join(outputdir, 'v1', 'dump_names', crash_id)
-    create_dir_if_needed(os.path.dirname(fn))
-    with open(fn, 'w') as fp:
-        json.dump(dumps.keys(), fp)
+    if fetchdumps:
```

Comment:
Mmm... I'm not sure this is worth the effort in this specific case, but I can do it.

Reply:
This is somewhat complicated because we need `dump_checksums` from the raw crash data to know which dumps to get so these two tasks are intertwined. It's definitely doable, but I think the end result is just going to be a more complicated form of the existing tangling. So I'm going to pass on this for now. If this command gets more complicated, we can revisit it then.